### PR TITLE
Feat / Create New Playlists 🌱🎚️

### PR DESCRIPTION
**Pull request by johnxgaehner**
- new page: CREATE NEW PLAYLISTS
- our playlist overview page is empty so we need a space where we can create playlists
- click on "Create New Playlist", fill out all inputs and submit to save the playlist in local storage
- you are also able to reset the inputs or go back to the playlist overview page

I already created a few playlist for possible upcoming gigs, next step is to implement a function where we can add tracks to our newly created playlists.

### CODE REVIEW

**Review by adrianammt — COMMENTED** [review]
It does everything you explained it would do. At first I didn't get that Cancel was Reset. But I guess that is a niche user thing. Great job!

File: src/pages/CreateNewPlaylistPage.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import { useState } from "react";
+import "./CreateNewPlaylistPage.css";
+import saveInLocalStorage from "../services/saveInLocalStorage";
```
[the remark this answers is not in the document]
Comment (johnxgaehner) [reply src/pages/CreateNewPlaylistPage.js:3]: it feels a bit like being a super dev when you import that function on another page! can only recommend! haha

[the remark this answers is not in the document]
Comment (namirsab) [reply src/pages/CreateNewPlaylistPage.js:3]: Just to clarify Adriana, this is not a hook, just a function. A hook is a react thing, that is implemented using a javascript function. But i also like it john, well done ;)

File: src/pages/CreateNewPlaylistPage.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+export default function CreateNewPlaylistPage() {
+  const [newPlaylist, setNewPlaylist] = useState({
+    playlistName: "",
+    playlistDescription: "",
+  });
+
+  function handleOnChange(event) {
+    const key = event.target.name;
+    const input = event.target.value;
+    const newPlaylistData = { ...newPlaylist, id: uuidv4(), [key]: input };
```
Comment (namirsab) [inline src/pages/CreateNewPlaylistPage.js:16]: I think you could just create the `id` on the `handleSubmit` function. Right now you are unnecessarily creating an `id` one every change.

File: src/pages/CreateNewPlaylistPage.css [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+.CNPP__row input {
+  all: unset;
+  width: 100%;
+}
+
+.CNPP__row button {
+  all: unset;
+}
```
Comment (namirsab) [inline src/pages/CreateNewPlaylistPage.css:26]: I see you have global css for all the `input` elements to set the focus. But I also see you are repeating this `unset` quite a lot. If all inputs are going to be the same, maybe you could just create an input component with its own css file, and then just reuse it everywhere. That way you wouldn't need to be doing `unset` in every page.

**Review by namirsab — APPROVED** [review]
Good job! Just a few comments :)

File: src/pages/CreateNewPlaylistPage.js (hunk 2 above)
Comment (johnxgaehner) [reply src/pages/CreateNewPlaylistPage.js:16]: ouh right, will try it out.

File: src/pages/CreateNewPlaylistPage.css (hunk 3 above)
Comment (johnxgaehner) [reply src/pages/CreateNewPlaylistPage.css:26]: tmrw it's definitely time to clean up! thx for mentioning it. :)

**Pull request merged**